lib/smoochApiBot.js

```diff
@@ -22,6 +22,23 @@ class SmoochApiBot extends Bot {
         });
         return api.appUsers.sendMessage(this.userId, message);
     }
+
+    setTypingActivity(typing) {
+        if(typing == undefined) {
+            typing = true;
+        }
+
+        const api = this.store.getApi();
+        let message = Object.assign({
```

Comment:
`message` should be renamed to `activity`,

Should also be a `const` not a `let`, even though the existing code isn't rigorously following this rule :]